src/py/flwr/client/app.py

```diff
@@ -422,14 +450,45 @@ def _on_backoff(retry_state: RetryState) -> None:
 
                     # Handle app loading and task message
                     try:
-                        # Load ClientApp instance
                         run: Run = runs[run_id]
-                        client_app: ClientApp = load_client_app_fn(
-                            run.fab_id, run.fab_version
-                        )
+                        if isolate and supernode_address is not None:
+                            # Generate SuperNode token
+                            token: int = generate_rand_int_from_bytes(RUN_ID_NUM_BYTES)
+
+                            # Share Message and Context with servicer
+                            clientappio_servicer.set_inputs(
+                                ClientAppIoInputs(
+                                    message=message,
+                                    context=context,
+                                    run=run,
+                                    token=token,
+                                )
+                            )
+                            # Run ClientApp
+                            verbose = True
+                            command = [
+                                "flwr-clientapp",
+                                "--address",
```

Comment:
Should this be `--supernode`? We use `--superlink` in `flower-supernode`

Reply:
Removed. Both `stdout` and `stderr` are set to None